pubsub/subscriptions/main.go

```diff
@@ -167,6 +170,37 @@ func pullMsgsSettings(client *pubsub.Client, subName string) error {
 	return nil
 }
 
+func pullMsgsConcurrencyControl(client *pubsub.Client, subName string, numGoroutines int) error {
+	ctx := context.Background()
+	// [START pubsub_subscriber_concurrency_control]
+	sub := client.Subscription(subName)
+	sub.ReceiveSettings.NumGoroutines = numGoroutines
```

Comment:
Does `numGoroutines` need to be an argument? Might be easier to understand if this uses a number directly.

(If that conflicts with samples in other languages, it's fine to leave it as-is.)